### ISSUE TITLE
An exception has been thrown during the rendering of a template: Warning: Undefined array key "timestamp_rfc3339"

### ISSUE THREAD

**Issue by mamchyts** [symfony/symfony#42622]
**Symfony version(s) affected**: 6.0.x-dev ("symfony/framework-bundle": "6.0.x-dev as 5.4.x-dev")

**Description**
Error in WebProfilerBundle when I try to open panel logger

**Additional context**
Full trace:

```
{
    "type": "runtime_error.An exception has been thrown during the rendering of a template (\"Warning: Undefined array key \"timestamp_rfc3339\"\").",
    "debug": {
        "trace": "#0 /var/www/html/vendor/twig/twig/src/Extension/CoreExtension.php(1544): Symfony\\Component\\HttpKernel\\DataCollector\\LoggerDataCollector->getProcessedLogs()
        #1 /var/www/html/var/cache/dev/twig/ff/fffd13b5499d19ce3159fd076d01c6c396402847f940bfff4730d00f74d8e782.php(203): twig_get_attribute(Object(Twig\\Environment), Object(Twig\\Source), Object(Symfony\\Component\\HttpKernel\\DataCollector\\LoggerDataCollector), 'processedLogs', Array, 'any', false, false, false, 49)
        #2 /var/www/html/vendor/twig/twig/src/Template.php(171): __TwigTemplate_4a76dc145693c52f2d54618a4e93e4a8c32480c08e1d7d585a6bdcb9bd109e6e->block_panel(Array, Array)
        #3 /var/www/html/var/cache/dev/twig/0e/0e3e7afe8b22fadbf5008ba6986a65873e5c3431be6357c43eac9384371261e3.php(214): Twig\\Template->displayBlock('panel', Array, Array)
        #4 /var/www/html/vendor/twig/twig/src/Template.php(171): __TwigTemplate_eb5369703c33bded5ad26b4230966824fe328c63e3b5af5526fda634eec81831->block_body(Array, Array)
        #5 /var/www/html/var/cache/dev/twig/73/736fd4d1c1ecc8d1642f11f2814611564f45ee01280f7cad29bd252ba27589d3.php(84): Twig\\Template->displayBlock('body', Array, Array)
        #6 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_0451b7c0a4d851b029ef09d4b4f0981b47d0041a0962b23e3e23dbbe0c412df0->doDisplay(Array, Array)
        #7 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\\Template->displayWithErrorHandling(Array, Array)
        #8 /var/www/html/var/cache/dev/twig/0e/0e3e7afe8b22fadbf5008ba6986a65873e5c3431be6357c43eac9384371261e3.php(50): Twig\\Template->display(Array, Array)
        #9 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_eb5369703c33bded5ad26b4230966824fe328c63e3b5af5526fda634eec81831->doDisplay(Array, Array)
        #10 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\\Template->displayWithErrorHandling(Array, Array)
        #11 /var/www/html/var/cache/dev/twig/ff/fffd13b5499d19ce3159fd076d01c6c396402847f940bfff4730d00f74d8e782.php(53): Twig\\Template->display(Array, Array)
        #12 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_4a76dc145693c52f2d54618a4e93e4a8c32480c08e1d7d585a6bdcb9bd109e6e->doDisplay(Array, Array)
        #13 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\\Template->displayWithErrorHandling(Array, Array)
        #14 /var/www/html/vendor/twig/twig/src/Template.php(379): Twig\\Template->display(Array)
        #15 /var/www/html/vendor/twig/twig/src/TemplateWrapper.php(40): Twig\\Template->render(Array, Array)
        #16 /var/www/html/vendor/twig/twig/src/Environment.php(277): Twig\\TemplateWrapper->render(Array)
        #17 /var/www/html/vendor/symfony/web-profiler-bundle/Controller/ProfilerController.php(117): Twig\\Environment->render('@WebProfiler/Co...', Array)
        #18 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(156): Symfony\\Bundle\\WebProfilerBundle\\Controller\\ProfilerController->panelAction(Object(Symfony\\Component\\HttpFoundation\\Request), '738961')
        #19 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(78): Symfony\\Component\\HttpKernel\\HttpKernel->handleRaw(Object(Symfony\\Component\\HttpFoundation\\Request), 1)
        #20 /var/www/html/vendor/symfony/http-kernel/Kernel.php(199): Symfony\\Component\\HttpKernel\\HttpKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)
        #21 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(37): Symfony\\Component\\HttpKernel\\Kernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request))
        #22 /var/www/html/vendor/autoload_runtime.php(29): Symfony\\Component\\Runtime\\Runner\\Symfony\\HttpKernelRunner->run()
        #23 /var/www/html/public/index.php(8): require_once('/var/www/html/v...')
        #24 {main}"
    }
}
```

Look like was broken in this PR: https://github.com/symfony/http-kernel/commit/d4b2daf421d867c6a9195cf91fbfe689ac2ad5aa

**Comment by ro0NL** [comment]
see #42600

**Comment by mamchyts** [comment]
Excellent. Thanks

**Comment by mamchyts** [comment]
Still not fixed.

Composer version:
```
        {
            "name": "symfony/http-kernel",
            "version": "6.0.x-dev",
            "source": {
                "type": "git",
                "url": "https://github.com/symfony/http-kernel.git",
                "reference": "5b5c09e35e6f926af8ed31041b4a550a820aea26"
            },
            "dist": {
                "type": "zip",
                "url": "https://api.github.com/repos/symfony/http-kernel/zipball/5b5c09e35e6f926af8ed31041b4a550a820aea26",
                "reference": "5b5c09e35e6f926af8ed31041b4a550a820aea26",
                "shasum": ""
            },
```






Full error text:
```
{
    "type": "runtime_error.An exception has been thrown during the rendering of a template (\"Warning: Undefined array key \"timestamp_rfc3339\"\").",
    "debug": {
        "trace": "#0 /var/www/html/vendor/twig/twig/src/Extension/CoreExtension.php(1544): Symfony\\Component\\HttpKernel\\DataCollector\\LoggerDataCollector->getProcessedLogs()
        #1 /var/www/html/var/cache/dev/twig/ff/fffd13b5499d19ce3159fd076d01c6c396402847f940bfff4730d00f74d8e782.php(203): twig_get_attribute(Object(Twig\\Environment), Object(Twig\\Source), Object(Symfony\\Component\\HttpKernel\\DataCollector\\LoggerDataCollector), 'processedLogs', Array, 'any', false, false, false, 49)
        #2 /var/www/html/vendor/twig/twig/src/Template.php(171): __TwigTemplate_4a76dc145693c52f2d54618a4e93e4a8c32480c08e1d7d585a6bdcb9bd109e6e->block_panel(Array, Array)
        #3 /var/www/html/var/cache/dev/twig/0e/0e3e7afe8b22fadbf5008ba6986a65873e5c3431be6357c43eac9384371261e3.php(214): Twig\\Template->displayBlock('panel', Array, Array)
        #4 /var/www/html/vendor/twig/twig/src/Template.php(171): __TwigTemplate_eb5369703c33bded5ad26b4230966824fe328c63e3b5af5526fda634eec81831->block_body(Array, Array)
        #5 /var/www/html/var/cache/dev/twig/73/736fd4d1c1ecc8d1642f11f2814611564f45ee01280f7cad29bd252ba27589d3.php(84): Twig\\Template->displayBlock('body', Array, Array)
        #6 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_0451b7c0a4d851b029ef09d4b4f0981b47d0041a0962b23e3e23dbbe0c412df0->doDisplay(Array, Array)
        #7 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\\Template->displayWithErrorHandling(Array, Array)
        #8 /var/www/html/var/cache/dev/twig/0e/0e3e7afe8b22fadbf5008ba6986a65873e5c3431be6357c43eac9384371261e3.php(50): Twig\\Template->display(Array, Array)
        #9 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_eb5369703c33bded5ad26b4230966824fe328c63e3b5af5526fda634eec81831->doDisplay(Array, Array)
        #10 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\\Template->displayWithErrorHandling(Array, Array)
        #11 /var/www/html/var/cache/dev/twig/ff/fffd13b5499d19ce3159fd076d01c6c396402847f940bfff4730d00f74d8e782.php(53): Twig\\Template->display(Array, Array)
        #12 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_4a76dc145693c52f2d54618a4e93e4a8c32480c08e1d7d585a6bdcb9bd109e6e->doDisplay(Array, Array)
        #13 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\\Template->displayWithErrorHandling(Array, Array)
        #14 /var/www/html/vendor/twig/twig/src/Template.php(379): Twig\\Template->display(Array)
        #15 /var/www/html/vendor/twig/twig/src/TemplateWrapper.php(40): Twig\\Template->render(Array, Array)
        #16 /var/www/html/vendor/twig/twig/src/Environment.php(277): Twig\\TemplateWrapper->render(Array)
        #17 /var/www/html/vendor/symfony/web-profiler-bundle/Controller/ProfilerController.php(117): Twig\\Environment->render('@WebProfiler/Co...', Array)
        #18 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(156): Symfony\\Bundle\\WebProfilerBundle\\Controller\\ProfilerController->panelAction(Object(Symfony\\Component\\HttpFoundation\\Request), '70e9c7')
        #19 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(78): Symfony\\Component\\HttpKernel\\HttpKernel->handleRaw(Object(Symfony\\Component\\HttpFoundation\\Request), 1)
        #20 /var/www/html/vendor/symfony/http-kernel/Kernel.php(199): Symfony\\Component\\HttpKernel\\HttpKernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request), 1, true)
        #21 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(37): Symfony\\Component\\HttpKernel\\Kernel->handle(Object(Symfony\\Component\\HttpFoundation\\Request))
        #22 /var/www/html/vendor/autoload_runtime.php(29): Symfony\\Component\\Runtime\\Runner\\Symfony\\HttpKernelRunner->run()
        #23 /var/www/html/public/index.php(8): require_once('/var/www/html/v...')
        #24 {main}"
    }
}
```

**Comment by ro0NL** [comment]
Does updating `symfony/monolog-bridge` solve it? If so, could you add a Composer conflict rule?

**Comment by mamchyts** [comment]
Thank you.

Next code fix my issue:
```composer req symfony/monolog-bridge:"6.0.x-dev as 5.4.x-dev"```

**Comment by stof** [comment]
Well, the fix has been merged in the 5.4.x branch (the lowest affected branch), but it has not yet been merged up to 6.0.x
